src/styles/blocks/day.scss

```diff
@@ -0,0 +1,25 @@
+.day {
```

Comment:
will be better to make it the element of the calendar block

Reply:
Done.